app/models/invenio_rdm_record_converter.rb

```diff
@@ -220,16 +220,23 @@ def build_creator_contributor_json(creator)
     json
   end # build_creator_contributor_json
 
-  def additional_titles(titles)
-    additional_titles_size = titles.size-1
-    return nil if additional_titles_size < 0
-    titles.last(additional_titles_size).map{ |title| {"title": title, "type": "alternative_title", "lang": ENG} }
+  def additional(category:, array:)
+    type = set_type(category)
+
+    # return all values except the first, remove the strings that contain only
+    # spaces or are empty, then map
+    array.slice(1..-1).delete_if(&:blank?).map{ |word|
```

Comment:
Just curious: `array.slice(1..-1)` versus `array[1..]` here? I think it's because of wanting the delete to run on a copy of the original... What about using [`filter_map`](https://ruby-doc.org/core-2.7.0/Enumerable.html#filter_map-method) instead?

Reply:
Actually, no difference 😅, but you do have to specify the end. So `array[1..-1]` would have worked. 
Wish I knew to use `filter_map`, it will work nicely. Maybe I'll add it later.

Reply:
ah maybe I had a different version of Ruby:
```
> irb
irb(main):001:0> a = [1,2,3]
irb(main):002:0> a[1..]
=> [2, 3]
```

```
> irb --version
irb 1.2.1 (2019-12-24)
> ruby --version
ruby 2.7.0p0 (2019-12-25 revision 647ee6f091) [x86_64-linux-gnu]
```